lib/index.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var configValidation, crypto, EventEmitter, mysql, timers;
+
+crypto = require("crypto");
+EventEmitter = require("events");
+mysql = require("mysql");
+timers = require("timers");
+configValidation = require("./config-validation")();
+module.exports = (backendConfig) => {
+    return require("./ddq-backend-mysql")(backendConfig, configValidation, crypto, EventEmitter, mysql, timers);
```

Comment:
We're making an instance.  How about you pass the config to the instance?

    module.exports = require("./ddq-backend-mysql)(configValidation, crypto, EventEmitter, mysql, timers);

Then inside the module your constructor looks like

    class DdqBackendMySql extends EventEmitter {
        constructor(backendConfig) {